src/client/conn/mod.rs

```diff
@@ -5,8 +5,8 @@
 //! are not handled at this level. This module provides the building blocks to
 //! customize those things externally.
 //!
-//! If don't have need to manage connections yourself, consider using the
-//! higher-level [Client](super) API.
+//! If you are looking for a convenient HTTP client,
+//! then you may wish to consider [reqwest](https://github.com/seanmonstar/reqwest).
```

Comment:
I think we might want to point to `hyper-util` instead of `reqwest`

Reply:
That’s not a convenient / high level http client though?

Reply:
It's higher level than this, I'd say we can just point to both of them here.

Reply:
```suggestion
//! then you may wish to consider [reqwest](https://github.com/seanmonstar/reqwest)
//! for a high level client or [`hyper-util`'s client](https://docs.rs/hyper-util/latest/hyper_util/client/index.html)
//! if you want to keep it more low level / basic.
```

Like this @dswij ?